### PR TITLE
Added few questions to enhance skills in knowledge

### PR DESCRIPTION
**Pull request by saurabhjn76**
progress in #119 
Changes: added question in knowledge

**Knowledge**

- When_is_question.txt

- acronyms .txt

- atomic_mass.txt

- capital.txt
- ceo.txt
- countries_information,txt
- time_to_travel_between_cities.txt
- wikipedia.txt.
- who.txt
- universities.txt

### CODE REVIEW

**Comment by saurabhjn76** [comment]
@DravitLochan @dynamitechetan @jyoth1raditya please review

**Review by DravitLochan — APPROVED** [review]
Good job done

**Review by Manveer695 — APPROVED** [review]
great!

File: models/general/knowledge/en/countries_information.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ eol
 
 #Tells us about time zone of different countries
 
-Time Zone of *|What is time zone of *| * Time zone 
+Time Zone of *|What is time zone of *| * Time zone|tell me the time zone of *
```
Comment (AliAyub007) [inline models/general/knowledge/en/countries_information.txt]: Add 
Do you know question here also and also if you write anything with article i.e tell me the time one of * then we should also make a query without "the" also

File: models/general/knowledge/en/capital.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Finds the capital of a country
-capital of *| what is the capital of *|capital *
+capital of *| what is the capital of *|capital *|tell me the capital of *|do you know the capital of *| search for the capital of *| find the capital of *| tell the capital of *| serach the capital of *|* capital
```
Comment (AliAyub007) [inline models/general/knowledge/en/capital.txt]: search typo in second last query

**Comment by saurabhjn76** [comment]
@AliAyub007 thanks, made the suggested changes

**Review by AliAyub007 — APPROVED** [review]

**Comment by hpdang** [comment]
@saurabhjn76 thank you. Please keep on updating the questions. I saw there is a wikipedia skill. There must be more things we can do with that. Please create an issue to extend the wikipedia skill. Find out what are other interesting answers/facts we can generate from that API. thanks

**Comment by saurabhjn76** [comment]
@hpdang Yes, I will keep adding more questions to the skill. Thanks :)

**Pull request merged**